script.js

```diff
@@ -75,6 +80,32 @@ const objectRateFetcher = (valFrom, valTo) => {
     }
   }
 };
+//-----------------------GRID--------
+
+const updateGrid = () => {
+  grid.innerHTML = "";
+  gridContainer.style.visibility = "visible";
+  gridContainer.style.opacity = "1";
+  gridContainer.style.transition = "all 0.5s";
+
+  for (let i = 0; i < currencyData.length; i++) {
+    if (currencyData[i].base === countriesFromSelect.value) {
+      //console.log("base" + currencyData[i].base);
+
+      for (let j = 0; j < Object.keys(currencyData[i].rates).length; j++) {
```

Comment:
how about you switched that to array functions instead? 👀

Reply:
done!